test_live_claim_fix.py

```
@@ -0,0 +1,128 @@
+1. A claim is bound to a writer with pid+boot_id
+2. A loser with a tiny poll budget does NOT unlink the claim when the owner is live
+3. The loser does NOT return different bytes (adopts the winner's bytes)
+"""
+import os
+import time
+import threading
+from pathlib import Path
+import tempfile
+import sys
+
+sys.path.insert(0, '/tmp/exec-tsk-lve2bo')
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp/exec-tsk-lve2bo"

(S108)